Capstone Project/PFObjects/Exercise.m

```diff
@@ -27,11 +27,11 @@ +(void) newExercise: (NSString *_Nullable)name withWeight:(float)weight withSet1
     Exercise *newExercise = [Exercise new];
     newExercise.name = name;
     newExercise.weight = weight;
-    newExercise.set1 = set1;
-    newExercise.set2 = set2;
-    newExercise.set3 = set3;
-    newExercise.set4 = set4;
-    newExercise.set5 = set5;
+    newExercise.exerciseSet1 = set1;
```

Comment:
This can be an array or a formatted string (x/x/x/x) stored the same way in db too. Only when it needs to be rendered in UI as labels, it can use '/' delimiter API to load in an array and set on labels.